scripts/package-build/linux-kernel/build-accel-ppp-ng.sh

```
@@ -18,10 +18,16 @@ if [ ! -f ${KERNEL_VAR_FILE} ]; then
 fi
 
-# Build VPP as we need VPP libraries
-cd ../vpp/
-./build.py
-cd ${CWD}
+# Build VPP as we need VPP libraries. This is a full VPP build (make pkg-deb),
+# not just headers, since that's the only build target VPP exposes here - skip
+# it if a previous run already produced the libraries we link against. Remove
+# ${VPP_LIB_CHECK_PATH} (or the whole ../vpp/vpp checkout) to force a rebuild,
+# e.g. after bumping the VPP commit_id in ../vpp/package.toml.
+if [ ! -d ${VPP_LIB_CHECK_PATH} ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)